deb/libsoletta/control

```diff
@@ -0,0 +1,34 @@
+Source: libsoletta
+Priority: optional
+Maintainer: Anselmo L. S. Melo <anselmo.melo@intel.com>
+Build-Depends: debhelper (>= 8.0.0), cdbs, python3 (>=3.4), chrpath, python3-jsonschema, libudev-dev, libpcre3-dev, libicu-dev, libcurl4-openssl-dev
```

Comment:
We shouldn't compile with sanitizer or use libasan

Em sábado, 12 de setembro de 2015, Anselmo L. S. Melo <
notifications@github.com> escreveu:

> In deb/libsoletta/control
> <https://github.com/solettaproject/soletta-packaging/pull/5#discussion_r39339581>
> :
>
> > @@ -0,0 +1,34 @@
> > +Source: libsoletta
> > +Priority: optional
> > +Maintainer: Anselmo L. S. Melo <anselmo.melo@intel.com <javascript:_e(%7B%7D,'cvml','anselmo.melo@intel.com');>>
> > +Build-Depends: debhelper (>= 8.0.0), cdbs, python3 (>=3.4), chrpath, python3-jsonschema, libudev-dev, libpcre3-dev, libicu-dev, libcurl4-openssl-dev
>
> libasan0 and libasan1 are missing here. both are installed together with
> gcc, so testing the pkgs on Ubuntu/debian with gcc hode this issue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta-packaging/pull/5/files#r39339581>
> .
>


-- 
Gustavo Sverzut Barbieri
--------------------------------------
Mobile: +55 (19) 99225-2202
Contact: http://www.gustavobarbieri.com.br/contact